src/parser.ts

```diff
@@ -164,82 +179,171 @@ class Parser {
-            
+
             const propTypeString = this.checker.typeToString(propType);
 
             const isOptional = (prop.getFlags() & ts.SymbolFlags.Optional) !== 0;
 
 
             const jsDocComment = this.findDocComment(prop);
 
+            let defaultValue = null;
+
+            if (defaultProps[propName] !== undefined) {
+                defaultValue = { value: defaultProps[propName] };
+            } else if (jsDocComment.tags.default) {
+                defaultValue = { value: jsDocComment.tags.default };
+            }
+
             result[propName] = {
                 required: !isOptional,
                 type: { name: propTypeString },
-                description: jsDocComment,
-
-                // TODO
-                defaultValue: null
+                description: jsDocComment.fullComment,
+                defaultValue
             };
         });
 
         return result;
     }
 
-    findDocComment(symbol: ts.Symbol) {
+    findDocComment(symbol: ts.Symbol): JSDoc {
         const comment = this.getFullJsDocComment(symbol);
-        if (comment) {
+        if (comment.fullComment) {
             return comment;
         }
 
         const rootSymbols = this.checker.getRootSymbols(symbol);
         const commentsOnRootSymbols = rootSymbols
             .filter(x => x !== symbol)
             .map(x => this.getFullJsDocComment(x))
-            .filter(x => !!x);
+            .filter(x => !!x.fullComment);
 
         if (commentsOnRootSymbols.length) {
             return commentsOnRootSymbols[0];
         }
 
-        return '';
+        return defaultJSDoc;
     }
 
     /**
      * Extracts a full JsDoc comment from a symbol, even
-     * thought TypeScript has broken down the JsDoc comment into plain
+     * though TypeScript has broken down the JsDoc comment into plain
      * text and JsDoc tags.
      */
-    getFullJsDocComment(symbol: ts.Symbol) {
+    getFullJsDocComment(symbol: ts.Symbol): JSDoc {
 
         // in some cases this can be undefined (Pick<Type, 'prop1'|'prop2'>)
         if (symbol.getDocumentationComment === undefined) {
-            return "";
+            return defaultJSDoc;
         }
 
         const mainComment = ts.displayPartsToString(symbol.getDocumentationComment());
 
         const tags = symbol.getJsDocTags() || [];
-        const tagComments = tags.map(t => {
-            let result = '@' + t.name;
-            if (t.text) {
-                result += ' ' + t.text;
-            }
-            return result;
+
+        const tagComments: string[] = [];
+        const tagMap: StringIndexedObject<string> = {};
+
+        tags.forEach(tag => {
+            const trimmedText = (tag.text || '').trim();
+            const currentValue = tagMap[tag.name];
+            tagMap[tag.name] = currentValue ? currentValue + '\n' + trimmedText : trimmedText;
+
+            if (tag.name !== 'default') { tagComments.push(formatTag(tag)); }
```

Comment:
i figured we can avoid adding the `@default whatever` to the description, as it's already represented by the `defaultValue`